src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -117,6 +117,13 @@ private function executeCallback(callable $callback, ContainerConfigurator $cont
                 case self::class:
                     $arguments[] = $this;
                     break;
+                case 'string':
```

Comment:
do we want to keep that case, or should we remove support for this argument in favor of the env() method, in order to have only one way to do this env-related check?

Reply:
I very much like this. To be able to write this is super cool: 

```php
return static function (FrameworkConfig $framework, string $env) {
```

However, the easier concept to understand is to do `ContainerConfigurator::env()` as you suggest. The `string $env` will only work "sometimes"...

```php
return static function (FrameworkConfig $framework, ContainerConfigurator $container) {
    $env = $container->env();
```

I'll remove this.